packages/convertkit/src/api/subscribe.ts

```diff
@@ -1,21 +1,35 @@
 import {NextApiRequest, NextApiResponse} from 'next'
 import {convertkitAxios} from '@skillrecordings/axios'
 import serverCookie from 'cookie'
+import {CONVERTKIT_SIGNUP_FORM, CONVERTKIT_TOKEN} from '@skillrecordings/config'
 
 const CK_SUBSCRIBER_KEY = process.env.NEXT_PUBLIC_CONVERTKIT_SUBSCRIBER_KEY
 const subscribe = async (req: NextApiRequest, res: NextApiResponse) => {
   if (req.method === 'POST') {
     try {
-      const {email_address, first_name, fields, tag} = req.body
-      const url = tag
-        ? `/tags/${tag}/subscribe`
-        : `/forms/${process.env.NEXT_PUBLIC_CONVERTKIT_SIGNUP_FORM}/subscribe`
+      const {
+        email_address,
+        first_name,
+        fields,
+        form = CONVERTKIT_SIGNUP_FORM,
+        tag,
+        sequence,
+      } = req.body
+      const getEndpoint = () => {
+        if (tag) {
+          return `/tags/${tag}/subscribe`
+        }
+        if (sequence) {
+          return `/sequences/${sequence}/subscribe`
+        }
+        return `/forms/${form}/subscribe`
+      }
```

Comment:
We can only do one of these while defaulting to standard form subscribe action. [ConvertKit API docs](https://developers.convertkit.com/#tag-a-subscriber) for reference.